.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest an enhancement for HTTPie
+title: ''
+labels: "new, feature"
+assignees: ''
+
+---
+
+**Checklist**
+
+- [ ] This is a new feature or functionality
+- [ ] This feature hasn't been proposed before
+
+___
+
+**Describe the feature or enhancement you'd like**
+A clear and concise description of what you want to happen
+
+**Is it related to a problem? Please describe**
+A clear and concise description of what the problem is. Eg. "I'm always frustrated when [...]". If there are any related issues, link them here
```

Comment:
I think it's better to avoid latinisms.  It also looks cleaner there's a line break after the quote.
```suggestion
A clear and concise description of what the problem is. For example: "I'm always frustrated when [...]".
If there are any related issues, link them here
```

Reply:
I agree that it looks cleaner on a new line, however, these section descriptions are meant to be deleted when a given user fills out a report. Therefore, it makes sense to keep it as concise as possible